recipes/anaconda-project/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "anaconda-project" %}
+{% set version = "0.7.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "f93cb98327465b9a400256851c20414a369fe285ee9015a69a85ed629fc7afed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Anaconda-Platform/anaconda-project.git
+  git_rev: v{{ version }}
```

Comment:
@jakirkham I know we prefer to use the releases, but the way this project is created, using the releases is not working correctly, so I need to use git source.